addons/product/models/product_packaging.py

```diff
@@ -77,4 +77,4 @@ def _compute_qty(self, qty, qty_uom=False):
         self.ensure_one()
         if qty_uom:
             qty = qty_uom._compute_quantity(qty, self.product_uom_id)
-        return float_round(qty / self.qty, precision_rounding=self.product_uom_id.rounding)
+        return float_round(qty / self.qty, precision_rounding=min(self.product_uom_id.rounding, 0.0001))
```

Comment:
a little odd to have absolute in my opinion, maybe either no rounding (if this is not considered as meant to be rounded), or have "self.product_uom_id.rounding / RATIO" (eg 100 in this case)